spec/unit/views/components/paginated_collection_spec.rb

```diff
@@ -140,23 +140,23 @@ def paginated_collection(*args)
       end
 
       it "should use 'post' as the collection name when there is no I18n translation" do
-        expect(pagination.find_by_class('pagination_information').first.content).to eq "Displaying <b>1</b> post"
+        expect(pagination.find_by_class('pagination-information').first.content).to eq "Displaying <b>1</b> post"
```

Comment:
Line is too long. [113/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.